npm/private/test/snapshots/bzlmod/npm_defs.bzl

```diff
@@ -745,238 +745,239 @@ load("@@_main~npm~npm__pify__4.0.1__links//:defs.bzl", store_741 = "npm_imported
 load("@@_main~npm~npm__pkg-dir__4.2.0__links//:defs.bzl", store_742 = "npm_imported_package_store")
 load("@@_main~npm~npm__pkg-types__1.0.2__links//:defs.bzl", store_743 = "npm_imported_package_store")
 load("@@_main~npm~npm__plotly.js__2.25.2__links//:defs.bzl", link_744 = "npm_link_imported_package_store", store_744 = "npm_imported_package_store")
-load("@@_main~npm~npm__point-in-polygon__1.1.0__links//:defs.bzl", store_745 = "npm_imported_package_store")
```

Comment:
I've been very tempted to try to reduce how much these change just from adding a single package :/